src/datatrove/tools/jobs_status.py

```diff
@@ -72,7 +76,7 @@ def main():
             emoji = "❌"
             incomplete_jobs += 1
 
-        if not (len(incomplete) == 0 and not args.show_complete):
+        if not (len(incomplete) == 0 and args.show_incomplete):
```

Comment:
Check comment above about `hide_complete`
```suggestion
        if len(incomplete) > 0 or not args.hide_complete:
```